src/Calendar.Plugin.Sample/SampleApp/ViewModels/CalendarRangePickerPopupViewModel.cs

```diff
@@ -9,78 +9,99 @@ namespace SampleApp.ViewModels
 {
     public class CalendarRangePickerPopupViewModel : BasePageViewModel
     {
-        public event Action<CalendarRangePickerResult> Closed;
+        private DateTime _maximumDate = DateTime.Today.AddYears(1);
 
-        public ICommand ClearCommand => new Command(() =>
-        {
-            SelectedDates = new List<DateTime> { 
-                DateTime.Today, 
-                DateTime.Today.AddDays(1),
-                DateTime.Today.AddDays(2)
-            };
-        });
+        private DateTime _minimumDate = DateTime.Today.AddYears(-1);
 
-        public ICommand SuccessCommand => new Command(async () =>
+        private DateTime _monthYear = DateTime.Today;
+
+        // If you do with _selectedDates then set _startDate & _endDate to null
+        private DateTime? _startDate = DateTime.Today.AddDays(-5);
+        private DateTime? _endDate = DateTime.Today.AddDays(5);
+
+        // If you do with startDate and endDate then set _selectedDates to null
+        private List<DateTime> _selectedDates = null;
+
+        // TODO create new class when we refactor sample app.
+        //private List<DateTime> _selectedDates = new List<DateTime> {
+        //    DateTime.Today.AddDays(1),
+        //    DateTime.Today.AddDays(2),
+        //    DateTime.Today.AddDays(3)
+        //};
+
+        public CalendarRangePickerPopupViewModel()
         {
-            Closed?.Invoke(new CalendarRangePickerResult()
-            {
-                IsSuccess = true,
-                SelectedDates = SelectedDates,
-            });
-            await PopupNavigation.Instance.PopAsync();
-        });
+            //SelectedDates = new List<DateTime>
```

Comment:
We should probably either remove these or uncomment them. Having them like this doesnt seem proper.